test/onnx/test_onnx_export.py

```diff
@@ -19,27 +20,66 @@
 sys.path.append(pytorch_test_dir)
 
 
+def export_to_onnx(
+    model: Union[torch.nn.Module, torch.jit.ScriptFunction],
+    input: Tuple[torch.Tensor],
+    custom_ops: Optional[List[Tuple[str, Callable, int]]] = None,
+    mocks: Optional[List[unittest.mock.patch]] = None,
```

Comment:
nit: I would use Sequence (as opposed to List) as a more general type. If order is not needed, we can use Iterable

Reply:
`Iterable` it is